src/main/kotlin/no/nav/familie/ba/sak/kjerne/brev/domene/ManueltBrevRequest.kt

```diff
@@ -64,7 +64,7 @@ data class ManueltBrevRequest(
 
     fun enhetNavn(): String = this.enhet?.enhetNavn ?: error("Finner ikke enhetsnavn på manuell brevrequest")
 
-    fun mottakerlandSED(): String =
+    fun mottakerlandSED(): List<String> =
         this.mottakerlandSed ?: error("Finner ikke mottakerland for SED på manuell brevrequest")
 }
```

Comment:
Siden mottakerlandSed ikke kan lenger være null så vil det som skjer etter ?: aldri skje.
Gitt at den defaulter til en tom liste, og så er det kanskje innafor å heller sjekke om lista er tom eller ikke?
ala

```suggestion
    fun mottakerlandSED(): List<String> =
        this.mottakerlandSed.takeIf { it.isNotEmpty() } ?: error("Finner ikke noen mottakerland for SED på manuell brevrequest")
```

Reply:
Ah digg! Hadde ikke sett så nøye på det etter at det kompilerte 😅 lagde commit og tok helg! Kjempebra forslag, tar det inn og skal teste det fullstack på mandag 😊